app/models/etsource/loader.rb

```diff
@@ -46,8 +46,11 @@ def presets
 
     def area_attributes(area_code)
       @area_attributes[area_code] ||= begin
-        area_attr = Atlas::Dataset.find(area_code).to_hash
+        area = Atlas::Dataset.find(area_code)
+        area_attr = area.to_hash
+        area_attr[:is_derived] = true if area.is_a?(Atlas::Dataset::DerivedDataset)
```

Comment:
> Also, I don't think it's right to have the is_ prefix; such prefixes are normally omitted in Ruby, especially in predicate methods.

Yes, I know - and that is one of the many things I love about Ruby.

I prepended `is_` here, because I am not defining a Ruby method. Instead I am filling a hash which is gonna be transferred as json. And in this we probably do not want to have a "?". There also is precedence for this: The countless `has_...` attributes of a dataset which could also be question-marked in Ruby.

But I guess you may as well argue the other way around and say it is fine to just leave out the prefix `is_` without adding the postfix "?". There is precedence for this as well, with the `useable` (virtual) attribute.
So I will remove the prefix!